src/answers-headless.ts

```diff
@@ -295,5 +296,31 @@ export default class AnswersHeadless {
     };
     this.stateManager.dispatchEvent('filters/toggleFacetOption', payload);
   }
+
+  addFilters(filterSetId: string, filters: SelectableFilter[]): void {
+    const payload = {
+      filterSetId,
+      filters
+    };
+    this.stateManager.dispatchEvent('filters/addFilters', payload);
+  }
+
+  selectFilterOption(filter: Filter, filterSetId?: string): void {
```

Comment:
Product mentioned in the Answers Headless feedback sheet that they would like us to move away from selectFacetOption and unselectFacetOption to an interface which has selected as a boolean param. For example: `setFacetOptionSelected(selected, fieldId, option)`. Personally I'm not a fan of boolean flags in functions because the behavior is less clear when reading the code, however as a middle ground, perhaps we could have a single `setFilter`  rather than `selectFilterOption` and `unselectFilterOption` which takes in a SelectableFilter rather than a Filter

Reply:
oh I see. In that case, I think we can just directly expose `toggleFilterOption` instead of wrapping it with `selectFilterOption` and `unselectFilterOption` to match what facet is doing. It would become like this:
`selectFilterOption(filter: SelectableFilter, filterSetId?: string): void`

Reply:
update: removed both selectFilterOption and unselectFilterOption. Now we only have `toggleFilterOption`